alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
```

Comment:
There is a syntax error in the import statement. The correct syntax should be `from logging.config import fileConfig` without any trailing slashes or other characters. Please correct this to avoid import errors.

Reply:
this automatic import